contents/docs/tutorials/feature-flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: How to Safely Roll Out New Features
```

Comment:
*Important*

I feel like we could make this more of a walkthrough ie:

How to safely roll out new features

* Define who to roll the feature out to
* Create a feature flag
* Implementing
  * Advanced controls
* Monitoring the changes made

The extra text underneath is valuable, but each could be its own entire section with more depth and more step-by-step

Reply:
@yakkomajuri thinking about it, I think this is probs the biggest improvement to be made, let this override the other comments IMHO

Reply:
So essentially the last section comes out (stored for a new page) and a new section comes in at the top about pre-flag creation, correct?

Reply:
Yes, with advanced controls also becoming a subset of implementation (as it's a detail not the main gist of what's being achieved).

# Idea

I think the reason the current set of titles is a little confusing is that it has a use case as the main tutorial topic "safely roll out features" as the title *then* has a bunch of use cases packed at the end, but they're quite technical sounding and they don't naturally connect with the tutorial unless you enable brain to translate from the feature to the use case.

How about we actually do:

* Define who to roll the feature out to
* Create a feature flag
* Implementing
* Advanced controls
* Using feature flags - include in here these use cases, and make the titles more like:
  * A/B testing -> "Checking impact on business metrics"
  * Gradual rollouts -> "Start with a few users first"
  * "Master Toggles" -> "Getting a huge feature into production"
  * Kill Switches -> "Enable easy rollbacks"
* Monitoring the changes made can then get dropped as the above cover it